### DISCUSSION TITLE
Documentation for rp2 DMA class?

### DISCUSSION THREAD

**Discussion by mendenm** [micropython/micropython#13291] [RP2040 / Pico] [answered]
@dpgeorge @jimmo I have not found the documentation for the new, highly anticipated DMA class.  I can import it, but can't use it yet.  This will be a huge step forward for the rp2!  Have docs or example code been generated yet?

**Comment by mattytrentini** [comment, marked as answer]
It seems the documentation didn't _quite_ make it in for v1.22, the relevant PR is #7670. It would be great to get your feedback on the docs!

**Reply by mendenm** [reply]
Thanks!  I'll look it over.
